src/components/visualizations/chart/highChartsCreators.ts

```diff
@@ -52,31 +51,3 @@ export function getHighchartsOptions(chartOptions: IChartOptions, drillConfig: a
         getCustomizedConfiguration(chartOptions, config, drillConfig),
     );
 }
-
-export function isDataOfReasonableSize(chartData: any, limits: IChartLimits, isViewByTwoAttributes = false) {
```

Comment:
I think you should move this to the place where it is used. Because stuff you move to utils should be tested. And it is used once anyway